config/install/dropzonejs.settings.yml

```diff
@@ -1 +1 @@
-tmp_dir: ''
+upload_scheme: temporary
```

Comment:
The goal of the PR isn't so much to make the upload destination configurable as to make DropzoneJS's upload handler use the stream wrapper system, bringing it in line with the way Drupal handles files in all other circumstances. If people wanted to put the uploads somewhere other than the temp directory, they could just as easily change tmp_dir. :)

I'd be perfectly OK with hard-coding the use of the temporary stream wrapper, if you'd prefer, but it seems like it might be best to leave that configurable...just in case.